src/components/card/MyReservationCard.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from "next/image";
+import { HTMLAttributes } from "react";
+
+import { cn } from "@/utils/classNames";
+import { formatNumber } from "@/utils/common";
+
+interface MyReservationCardProps extends HTMLAttributes<HTMLDivElement> {
+  activity: {
+    bannerImageUrl: string;
+    title: string;
+  };
+  status: "pending" | "confirmed" | "declined" | "canceled" | "completed";
+  reviewSubmitted: boolean;
+  totalPrice: number;
+  headCount: number;
+  date: string;
+  startTime: string;
+  endTime: string;
+}
+
+/**
+ * 예약관리 리스트 아이템
+ */
+export default function MyReservationCard({
+  status,
+  activity,
+  date,
+  startTime,
+  endTime,
+  totalPrice,
+  headCount,
+  reviewSubmitted,
+  className = "",
+}: MyReservationCardProps) {
+  /* TODO: Button component 반영 */
+  function ReviewButtons() {
+    return reviewSubmitted ? (
+      <>
+        <button className="w-[100%]">예약 변경</button>
+        <button className="w-[100%]">예약 취소</button>
+      </>
+    ) : (
+      <button className="w-[100%]">후기 작성</button>
+    );
+  }
+  return (
+    <li className="grid gap-[20px]">
+      <hr className="border-gray-50 tablet:hidden" />
+      <div className="grid gap-[12px]">
+        <div className="text-16-b text-gray-800 pl-[8px] tablet:hidden">
+          {date}
+        </div>
+        <section
```

Comment:
```suggestion
        <div
```
`section`은 카드보다는 더 큰, 페이지 단위의 구조적인 의미로 사용되는 것이 더 알맞을 것 같아요 🤔